ft_defaults.m

```diff
@@ -212,6 +212,12 @@
     ft_hastoolbox('fileexchange', 3, 1); % not required
   end
 
+  try
```

Comment:
this should be implemented using ft_hastoolbox. The mayo_mef code should not be on the path of people don't use it